CHANGELOG

```diff
@@ -2,6 +2,37 @@
 
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.4.4] - 2023-07-08
+
+### Added
+- Add dependency descriptor stream tracker for svc codecs (#1788)
+- Full reconnect on publication mismatch on resume. (#1823)
+- Pacer interface in down stream path. (#1835)
+- retry egress on timeout/resource exhausted (#1852)
+
+### Fixed
+- Send Room metadata updates immediately after update (#1787)
+- Do not send ParticipantJoined webhook if connection was resumed (#1795)
+- Reduce memory leaks by avoiding references in closure. (#1809)
+- Honor bind address passed as `--bind` also for RTC ports (#1815)
+- Avoid dangling downtracks by always deleting them in receiver close. (#1842)
+- Better cleanup of subscriptions with needsCleanup. (#1845)
+- Fix nack issue for svc codecs (#1856)
+- Fixed hidden participant update were still sent when track is published (#1857)
+- Fixed Redis lockup when unlocking room with canceled request context (#1859)
+
+### Changed
+- Improvements to A/V sync (#1773 #1781 #1784 )
+- Improved probing to be less disruptive in low bandwidth scenarios (#1782 #1834 #1839)
+- Do not mute forwarder when paused to bandwidth congestion. (#1796)
```

Comment:
:nit: paused due to

Reply:
ah ok, i copied this from the commit log. will change

Reply:
Sorry, my hands most of the time do not keep up with my head and I mess up a lot😞